Makefile

```diff
@@ -27,7 +27,8 @@ frakti: $(shell $(LOCALKUBEFILES))
 .PHONY: docker
 docker:
 	cp ${BUILD_DIR}/flexvolume_driver deployment/flexvolume/
-	sudo docker build -t stackube/flex-volume:v0.1 deployment/flexvolume/
+	@if test -z "${IMAGE_VERSION}"; then echo "IMAGE_VERSION is not set"; exit 1; fi
```

Comment:
I think we should set a default version instead of reporting a error. The default should track the current version.

Reply:
fixed